src/components/evm/AppNav.vue

```diff
@@ -123,7 +123,7 @@ export default defineComponent({
             // if the user has come from an external source, pressing back should go to the parent route
             const navigatedFromApp = sessionStorage.getItem('navigatedFromApp');
 
-            if (navigatedFromApp) {
+            if (navigatedFromApp && this.$route.query.tab !== 'attributes') { // @TODO refactor to avoid explicit conditionals, required to nav back from details page but retain tab navigation history
```

Comment:
Since the 'back' functionality exists on the details page with subroutes (tabs), this is required to navigate back to the inventory page while retaining 'back' for the tab navigation.